### PR TITLE
switch to GitHub actions

### PR DESCRIPTION
**Pull request by dlockhart**

### CODE REVIEW

File: autocomplete.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { PolymerElement } from '@polymer/polymer/polymer-element.js';
 import { Debouncer } from '@polymer/polymer/lib/utils/debounce.js';
-import 'd2l-dropdown/d2l-dropdown-content.js';
-import 'd2l-dropdown/d2l-dropdown.js';
 import '@brightspace-ui/core/components/colors/colors.js';
+import '@brightspace-ui/core/components/dropdown/dropdown-content.js';
+import '@brightspace-ui/core/components/dropdown/dropdown.js';
```
Comment (dlockhart) [inline autocomplete.js:5]: Seemed like an easy fix to do at the same time -- these are directly aliased from those old d2l-dropdown files.

**Review by AdmiralCoco — APPROVED** [review]

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
```
Comment (svanherk) [inline .github/workflows/release.yml:5]: Should this have the other branches because it does publish to npm (I think I missed this in another PR today...)

Comment (dlockhart) [reply .github/workflows/release.yml:5]: I haven't been adding it for anything except core and Intl. I think it'll be suuuuuuper rare to ever care about release branches for these. I figure we can always add that support later? 🤷

Comment (svanherk) [reply .github/workflows/release.yml:5]: Yeah it's probably not common, I've had to do it a few times for facet-filter-sort but it doesn't publish to npm yet.  I think it depends how active development is on a labs component.  But we can definitely always add it later

**Pull request merged**